### PR TITLE
add encodeURIComponent to query results report

### PR DESCRIPTION
**Pull request by titoBouzout**

### CODE REVIEW

**Comment by LadyBluenotes** [comment]
Could you please add a description to explain this or link an issue that this is related to?

**Comment by titoBouzout** [comment]
Sure, sorry about that. 

When constructing links, the value of the parameters must be url encoded, else the receiver server could see garbled text when using non-english, and/or the text could be cut when using special characters.

For `quack & murci`, the difference between these links
(encoded)
`https://github.com/solidjs/solid-docs-next/issues/new?title=quack%20%26%20murci`
(not encoded)
`https://github.com/solidjs/solid-docs-next/issues/new?title=quack & murci`

when opening the second link, it will display "quack" instead of "quack & murci"

File: src/ui/search.tsx [hunk 1]
```diff
@@ -190,7 +190,7 @@ export function Search() {
 								<p class="!mb-3 !mt-4 text-center text-sm">
             Believe this query should return results?{" "}
 									<A
-										href={`https://github.com/solidjs/solid-docs-next/issues/new?title=[Search]+Missing+results+for+query+%22${searchTerm()}%22`}
+										href={`https://github.com/solidjs/solid-docs-next/issues/new?title=[Search]+Missing+results+for+query+%22${encodeURIComponent(searchTerm())}%22`}
```
Comment (atilafassina) [inline src/ui/search.tsx]:
```suggestion
										href={`https://github.com/solidjs/solid-docs/issues/new?title=[Search]+Missing+results+for+query+%22${encodeURIComponent(searchTerm())}%22`}
```

**Review by atilafassina — CHANGES_REQUESTED** [review]
Awesome!! Just a slight change since the repo moved.

**Review by atilafassina — APPROVED** [review]

**Pull request merged**